modules/front-end/src/app/core/components/member-drawer/member-drawer.component.ts

```diff
@@ -49,6 +49,13 @@ export class MemberDrawerComponent {
       policyId: [''],
       groupId: ['']
     });
+
+    this.memberForm.get('policyId').valueChanges.subscribe(() => {
+      this.validatePermissions();
+    })
+    this.memberForm.get('groupId').valueChanges.subscribe(() => {
+      this.validatePermissions();
+    })
   }
```

Comment:
Same bug as I comment under `access-token-drawer.component.ts`, I suggest removing these subscriptions, because `validatePermissions` will be called when we submit the form.

Reply:
Agree, made the required changes.